client/app/scripts/superdesk-authoring/metadata/views/metadata-dropdown.html

```diff
@@ -1,6 +1,7 @@
 <div class="dropdown" dropdown>
 	<div class="dropdown-toggle" dropdown-toggle>{{ item[field].name || item[field]}} </div>
 	<ul class="dropdown-menu pull-right">
-		<li ng-repeat="o in list" ng-click="select(o)"><button ng-disabled = "disabled">{{ o.name }}</button></li>
+		<li ng-click="select()" translate><button>None</button></li>
```

Comment:
Just to make sure - how does the `translate` directive work? Does it work with nested elements, too, or does it have to be directly applied to an element that contains the text node we want to translate?
(in other words - does it need to be moved onto the `<button>` element?)